module3/preparation/Week2/FeatureTesting.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: page
+title: Feature Testing
+---
+
+## Prework for [Feature Testing](/module3/lessons/Week2/FeatureTesting)
+
+1. Read [this article] about different types of testing.
```

Comment:
I think it more nuanced and should maybe come during a later mod, but this is basically my current philosophy on testing: https://kentcdodds.com/blog/static-vs-unit-vs-integration-vs-e2e-tests

Reply:
This article seems pretty good for unit/integration/e2e: https://www.twilio.com/blog/unit-integration-end-to-end-testing-difference

Reply:
haha - totally didn't realize that I didn't like the article.  I like the twilio one better than what I was going to use though - so happy accidents!